docs/examples/101/api-management-modular/NameValues.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "apimInstanceName": {
+      "type": "string",
+      "minLength": 1
+    }
+  },
+  "functions": [],
+  "variables": {
+    "apimNameValueSet": [
+      {
+        "displayName": "NameValue1",
+        "value": "SomeValue1",
+        "tags": [
+          "Example"
+        ],
+        "isSecret": false
+      },
+      {
+        "displayName": "NameSecretValue",
+        "value": "SomeSecretValue",
+        "tags": [
+          "Example",
+          "AnotherExampleTag"
+        ],
+        "isSecret": true
+      }
+    ]
+  },
+  "resources": [
+    {
+      "copy": {
+        "name": "apiManagementNVPair",
+        "count": "[length(variables('apimNameValueSet'))]"
+      },
+      "type": "Microsoft.ApiManagement/service/namedValues",
+      "apiVersion": "2020-06-01-preview",
+      "name": "[format('{0}/{1}', parameters('apimInstanceName'), variables('apimNameValueSet')[copyIndex()].displayName)]",
+      "properties": {
+        "displayName": "[variables('apimNameValueSet')[copyIndex()].displayName]",
+        "secret": "[variables('apimNameValueSet')[copyIndex()].isSecret]",
+        "value": "[variables('apimNameValueSet')[copyIndex()].value]",
+        "tags": "[variables('apimNameValueSet')[copyIndex()].tags]"
+      },
+      "dependsOn": []
+    }
+  ],
+  "outputs": {
+    "apimNameValues": {
+      "type": "array",
+      "copy": {
+        "count": "[length(variables('apimNameValueSet'))]",
+        "input": {
+          "nameValueId": "[resourceId('Microsoft.ApiManagement/service/namedValues', parameters('apimInstanceName'), variables('apimNameValueSet')[copyIndex()].displayName)]",
+          "nameValueName": "[variables('apimNameValueSet')[copyIndex()].displayName]"
+        }
+      }
+    }
+  },
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.3.126.58533",
```

Comment:
Did you generate these JSON files with the `SetBaselines.ps1` script? The `metadata._generator.version` property should be set to `dev`

This is causing a test to fail